sync/sdk.md

```diff
@@ -0,0 +1,81 @@
+# Sync SDK
```

Comment:
@danielpassos @wtrocki I'd appreciate and feedback or input into this early draft of the sync SDK and how the memeolist app could use it.
See @secondsun proposal for more info https://github.com/aerogear/proposals/pull/41